### PR TITLE
add "big" theme

### PR DESCRIPTION
**Pull request by nikolas**

### CODE REVIEW

**Comment by mcfiredrill** [comment]
guy_at_keyboard

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -22,6 +22,7 @@
         <% theme_basenames.each do |t| %>
           <%= link_to t, "#", class: 'set-theme', data: { theme: "#{t}" } %>
         <% end %>
+        <%= link_to 'big', 'https://bigchan.herokuapp.com/', target: 'blank' %>
```
Comment (mcfiredrill) [inline app/views/layouts/application.html.erb]: Should it be `_blank` instead?

Comment (nikolas) [inline app/views/layouts/application.html.erb]: yeah! :woops

**Comment by mcfiredrill** [comment]
big theme has been merged

**Pull request merged**